streaming/flink/helm-values/kube-prometheus.yaml

```diff
@@ -1,5 +1,9 @@
 prometheus:
   prometheusSpec:
+    serviceMonitorSelectorNilUsesHelmValues: false
+    podMonitorSelectorNilUsesHelmValues: false
+    serviceMonitorSelector: { }
```

Comment:
extra space between curly braces. Is that ok? one below as well

Reply:
Removed